development/lint/electron.js

```
@@ -19,3 +20,25 @@ try {
+  try {
+    const shellCommand = `grep '${expectedAppName}' "${distAppPath}"`;
+    console.log(shellCommand);
+    const grepResult = execSync(shellCommand, { encoding: 'utf-8' });
+    console.log('grepResult:', grepResult);
+    if (grepResult === '') {
+      throw new Error('grep command should not return output');
+    }
+  } catch (grepError) {
+    throw new Error(
+      `APP_NAME must be set to "OneKey Wallet" in the built app.js file. ` +
+      `Expected: ${expectedAppName}`
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)